src/store.js

```diff
@@ -0,0 +1,8 @@
+import { createStore, applyMiddleware } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import thunk from 'redux-thunk';
+import rootReducer from './reducers';
+
+const store = createStore(rootReducer, composeWithDevTools(applyMiddleware(thunk)));
+
+export default store;
```

Comment:
The redux store needed to be transferred to its own folder to prevent interference when I was trying to access store outside of React components.(in the Apollo requests)

Reply:
Interesting - hey as long as they both work!